source/api_documentation/transactions_api/_intro.rst

```diff
@@ -3,7 +3,7 @@
 Authentication
 --------------
 
-The API uses OAuth Authentication to authenticate that access is allowed. Pass the OAuth Token like any other query parameter, however, please note that the OAuth token is a required parameter.
+The API uses OAuth to validate access. The OAuth Token can be passed in two ways. The first way is to pass it like any other query parameter. The second is to pass the OAuth Token in the header of the request. Please note that the OAuth Token is a required parameter.
```

Comment:
should we flip the options around.. ie mention first way is to pass in header of request (I assume that will be a new preferred best practice as it keeps it out of the url for GET requests)

Reply:
Yeah, that's a good call. Let me flip this around. Thanks for catching that.

Reply:
done